conf/modules.config

```diff
@@ -229,6 +213,7 @@ process {
 
     withName: SAMTOOLS_VIEW_FOR_ARRIBA {
         ext.args = { "--output-fmt cram" }
+        ext.prefix = { "${meta.id}_star_for_arriba" }
```

Comment:
Out of the scope of this PR, but I wonder if we would want more general names (e.g. `alignment_for_arriba`) at some point. Not sure really

Reply:
I don't think we should change the name of the process, with how it is currently it is easy to understand which process was executed. We can change the name of the cram file to something else for sure

Reply:
let's leave it as it is for now :)